tests/e2e/features/redirection/mainPage/redirection.feature

```diff
@@ -1,4 +1,4 @@
-@redirections @regression @mainPage @authorized @smoke
+@redirection @regression @mainPage @authorized @smoke
```

Comment:
Are we going to use any tag from this list in e2e.yml? Probably this set is not callable now, is not it?